app/code/Magento/Checkout/Block/Checkout/AttributeMerger.php

```diff
@@ -280,10 +286,16 @@ protected function isFieldVisible($attributeCode, array $attributeConfig, array
      * @param array $attributeConfig
      * @param string $providerName name of the storage container used by UI component
      * @param string $dataScopePrefix
+     * @param array $additionalConfig
      * @return array
      */
-    protected function getMultilineFieldConfig($attributeCode, array $attributeConfig, $providerName, $dataScopePrefix)
-    {
+    protected function getMultilineFieldConfig(
```

Comment:
As pointed out by @sidolov, changes on protected methods are not allowed because it will make it backward incompatible. As described on the docs, the possible solutions is:

```
Instead of adding parameters to protected methods, Create a new method with a new signature and deprecate the old method without changing it.

Declare the new method as private if possible.
```
https://devdocs.magento.com/contributor-guide/backward-compatible-development/

Reply:
Hi @gabrieldagama thanks for your hint, code has been adjusted.